src/main/java/frc/robot/poseestimator/PoseEstimatorMath.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.poseestimator;
+
+import edu.wpi.first.math.geometry.*;
+
+public class PoseEstimatorMath {
+
+	public static double getKalmanRatio(double odometryStandardDeviation, double visionStandardDeviation) {
+		return odometryStandardDeviation / (odometryStandardDeviation + visionStandardDeviation);
+	}
+
+	public static Rotation2d combineVisionHeadingToGyro(
+		Rotation2d visionEstimatedHeading,
+		Rotation2d sampleDifferenceFromHeading,
```

Comment:
```suggestion
		Rotation2d changeInAngleSinceVisionObservationWasTaken,
```

I would move the calculation of it to the function body itself, since it's anyway computed using what anyways given to the function, and it matches its responsibility